Automation/Automation/Pages/AccountPage.cs

```diff
@@ -8,12 +9,39 @@ public class AccountPage
         #region Selectors
 
         private readonly By _welcomeText = By.CssSelector("p.hello strong");
-
+        private readonly By _editBillingAddress = By.XPath("//*[@id=\"top\"]/body/div/div[1]/div[2]/div/div[2]/div[2]/div/div[5]/div[2]/div[1]/div/div[1]/a");
```

Comment:
why did you use Xpath? There is no other option to make the selector look prettier?